packages/web3/src/contract/contract.ts

```diff
@@ -745,7 +745,7 @@ export class Contract extends Artifact {
     }
   }
 
-  private _fromApiContractState(state: node.ContractState): ContractState {
+  fromApiContractState(state: node.ContractState): ContractState {
```

Comment:
`fetchState(address: group)` should work with your fix now, and no need to make this public?

Reply:
It's useful in some cases, for example, in the bridge UI, when the user selects a token, we don't know whether the token is a local token or a wrapper for a remote token. Now in the bridge UI, we will first get the contract state by the `NodeProvider`, and get the contract type according to the code hash. So if this function is public, we don't need to re-fetch the contract state.